tools/topology/sof/pipe-passthrough-playback-variable.m4

```diff
@@ -0,0 +1,46 @@
+# Low Latency Passthrough Pipeline and PCM
```

Comment:
comment on file name "variable" is ambiguous.

It looks like there is no longer a dependency on sample rate, channels and format.

In Android HALs that was called 'dynamic' and defined by the user/application,

Reply:
I'm little bit concerned this will be confused to our "dynamic pipeline" feature if the name is "dynamic-"... I'm open to all naming, I tried "bt" -> not good, also "variable" seems not good. This doesn't do anything else compared to simple passthrough than set the channel count etc. ranges to allow more formats.

Reply:
@plbossart the diff to basic passthrough is:

< PCM_CAPABILITIES(Passthrough Playback PCM_ID, `S32_LE,S24_LE,S16_LE', PCM_MIN_RATE, PCM_MAX_RATE, 1, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)
---
> PCM_CAPABILITIES(Passthrough Playback PCM_ID, COMP_FORMAT_NAME(PIPELINE_FORMAT), PCM_MIN_RATE, PCM_MAX_RATE, 2, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)

So there's a fixed formats range (not defined by top level pipeline); we have this also in some other pipes. The top level pipeline channels is now defining "max channels" and min channels is hard coded to 1. We don't have currently means to convey the channel range from top level, it is always hardcoded from 2 -> 2. I need alternating channel count as in bt offload the dai config is stereo in a2dp and mono in sco.

So yes, by using this pipeline you have to little bit know what you are doing -> but it will be now used only in bt offload. I don't want to modify every single pipeline m4 macro to set the channel range as we have topology2 coming soon -> just doesn't make too much sense and is really error prone.